test/src/partner.c

```diff
@@ -198,7 +203,12 @@ int main(int argc, const char * argv[]) {
     if (staticTests)
         exit(EXIT_SUCCESS);
 
-    igs_start_with_device(p_networkDevice, port);
+
+    if (broker) {
+        brokerCommandSparing(broker);
```

Comment:
brokerCommandSparing is not part of the PR and it would be best to add all the broker-related code here directly.